website/docs/voor-maintainers/iconen.md

```diff
@@ -7,9 +7,16 @@ De DSO Toolkit biedt een iconenset aan. Deze set is op een aantal manieren te ge
 - Voeg icoon toe aan `/packages/dso-toolkit/src/icons`.
 - Bundel het icoon mee in de Web Component implementatie in `/packages/core/src/components/icon/icon.tsx`.
 
-## Spitesheet
+## Spritesheet
 
-Zowel de HTML/CSS implementatie als de `di` mixins maken gebruik van de spritesheet (`dso-icons.svg`). Deze spritesheet wordt gegenereerd aan de hand van de `.svg` en `.scss` files in `/packages/dso-toolkit/src/icons`.
+Zowel de HTML/CSS implementatie als de `di` mixins maken gebruik van de spritesheet (`dis.svg`). Deze spritesheet wordt gegenereerd aan de hand van de `.svg` en matchende (naam)`.scss` file in `/packages/dso-toolkit/src/icons`.
```

Comment:
Dit is niet meer waar: De HTML/CSS implementatie maakt gebruik van `dso-icons.svg`, de di mixins maken gebruik van `di.svg` (ik neem aan dat `dis.svg` een typefout is?)

En ik zou graag een onderbouwing willen zien waarom de lege selector nodig is, en wanneer deze niet meer nodig is.